src/View/mainpage/IsodoseTab.py

```diff
@@ -17,12 +19,29 @@ def __init__(self):
         self.color_squares = self.init_color_squares()
         self.checkboxes = self.init_checkboxes()
 
+        # Create and initialise ISO2ROI button and layout
+        self.iso2roi_button = QtWidgets.QPushButton()
+        self.iso2roi_button.setText("Convert Isodose to ROI")
```

Comment:
Since you convert many Isodoses to ROIs, should this be "Convert Isodoses to ROIs"? Same thing for the title in the Add-On Options

Reply:
Changes have been committed.